commandline/build.gradle

```diff
@@ -18,6 +18,8 @@ dependencies {
   implementation 'com.google.guava:guava'
   implementation 'org.apache.logging.log4j:log4j-api'
   implementation 'org.apache.logging.log4j:log4j-core'
+  implementation 'org.hyperledger.besu:plugin-api'
```

Comment:
why plugin-api? That seems ... strange ..

Reply:
The underlying type MetricCategory is defined in the plugin-api so we need both this and Besu metrics module unfortunately.